src/content_provider/sqlPreviewContentProvider.ts

```diff
@@ -83,10 +83,9 @@ export class SqlPreviewContentProvider
       await project.refreshProjectConfig();
       return await project.unsafeCompileQuery(query);
     } catch (error: any) {
-      window.showErrorMessage(
-        `Error while compiling: ${JSON.stringify(error)}`,
-      );
-      return error;
+      const errorMessage = (error as Error).message;
```

Comment:
WIll always have Message key?

Reply:
Yes we fixed that behaviour. Before we were sometimes rejecting promises with non Error objects.